config/isNull.js

```diff
@@ -0,0 +1,13 @@
+const isNull = (req, res, next) => {
+  if (req.body.email === '') {
+    req.body.email = null;
+  }
+  if (req.body.phone === '') {
+    req.body.phone = null;
+  }
+  next();
+};
+
+module.exports = {
+  isNull,
+};
```

Comment:
Do we need this anymore? You can add this logic to the sanitize function

Reply:
@herzizza88 we dont need this file anymore, right?

Reply:
ah yes ..have removed